fv3config/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 from ._config import (
     config_to_namelist, config_from_namelist, get_default_config,
-    write_run_directory,
+    write_run_directory, enable_restart
```

Comment:
Could you add a description to the documentation?

Reply:
Again, `enable_restart` was introduced in #9. But docs are needed. I added an issue #24 .

Reply:
I also added a Trello card. We should probably get in the habit of putting issues on Trello.